src/transformers/pipelines/token_classification.py

```diff
@@ -177,18 +179,14 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
                         input_ids = tokens["input_ids"].cpu().numpy()[0]
 
             score = np.exp(entities) / np.exp(entities).sum(-1, keepdims=True)
-            labels_idx = score.argmax(axis=-1)
```

Comment:
Because we are going to set the labels according to the strategy we need the scores for all labels, specially when using “average” strategy.